src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -117,6 +117,12 @@ func NewStartCommand(
 	command := &cobra.Command{
 		Use:   "start",
 		Short: "Start redpanda",
+		FParseErrWhitelist: cobra.FParseErrWhitelist{
```

Comment:
should we rename `whitelist -> allowlist`

Reply:
what's `fparse`

Reply:
It's from Cobra: https://github.com/spf13/cobra/blob/master/command.go#L32

Reply:
ah